index.js

```diff
@@ -49,20 +49,33 @@ Sevnup.prototype.loadAllKeys = function loadAllKeys() {
 };
 
 /**
+ * When you are done working on a key, or no longer want it within bookkeeping
+ * you can alert sevnup to forget it.  This notifies the ring that it doesn't
+ * need attention in the event this node goes down or hands off ownership.
+ * We want the service to be ignorant of vnodes so we rediscover the vnode.
+ * @param {string} key The key you have finished work on.
+ */
+Sevnup.prototype.workCompleteOnKey = function workCompleteOnKey(key) {
```

Comment:
yes, README has a couple things pending, this is just to race to keep up with IAIN